tensorrt_llm/_torch/auto_deploy/custom_ops/trtllm_dist.py

```diff
@@ -0,0 +1,119 @@
+"""TRT-LLM distributed operations and fused kernels.
+
+This module defines atomic TRT-LLM-specific ops that use optimized kernels.
+The torch fallback variants are defined separately to enable multi-pattern matching.
+"""
+
+from typing import List, Optional
+
+import torch
+
+from ..distributed.common import ReduceOp, get_rank_world_size, get_world_size, is_ompi
+
+# use trtllm distributed ops to improve TP performance if possible
+try:
```

Comment:
nit: we don't need a `try - except` block here. Please remove it. That is just legacy code when we were still a separate repo